ecmascript/transforms/base/src/hygiene/mod.rs

```diff
@@ -665,6 +665,21 @@ impl<'a> VisitMut for Hygiene<'a> {
         n.class.visit_mut_with(self);
     }
 
+    fn visit_mut_class_method(&mut self, n: &mut ClassMethod) {
+        let mut folder = Hygiene {
+            config: self.config.clone(),
+            current: Scope::new(ScopeKind::Fn, None),
+            ident_type: IdentType::Ref,
+            var_kind: None,
+        };
+
+        folder.ident_type = IdentType::Binding;
+        n.function.params.visit_mut_children_with(&mut folder);
+        folder.apply_ops(n);
```

Comment:
I think this can be changed to `folder.apply_ops(&mut n.params)`.

Reply:
I tried apply `n.function.params` as node, it makes test fail by identifier reference collision like

```
fn4(arg1, param, arg2, param1) {
        var opt1 = param.opt1, opt2 = param.opt2, opt3 = param.opt3, opt4 = param.opt4;
```

for newly added test cases. Updated to supply `n.function` instead.

Reply:
I see, thanks!